### DISCUSSION TITLE
Quasar v2  boot axios interceptors response no working

### DISCUSSION THREAD

**Discussion by darrentu8** [quasarframework/quasar#14327] [General - Components / Directives / etc]
### What happened?

// boot axios
instance.interceptors.response.use(
  function (response) {
    // Do something with response data
    return response;
  },
  function (error) {
    return Promise.reject(error);
  }
);

### What did you expect to happen?

"response " is no response 

### Reproduction URL

https://codesandbox.io/s/nifty-mountain-wmlbot?file=/src/boot/axios.js

### How to reproduce?

after basic init quasar2

### Flavour

Quasar CLI with Webpack (@quasar/cli | @quasar/app-webpack)

### Areas

Quasar CLI Commands/Configuration (@quasar/cli | @quasar/app-webpack | @quasar/app-vite), Plugins (quasar), Quasar Extras (@quasar/extras)

### Platforms/Browsers

Chrome

**Comment by pdanpdan** [comment]
Hello. This has nothing to do with quasar - please read axios docs on how to set interceptors.

The only thing related to quasar is the boot file, where:

- what is outside the function is only called once at start (first load - so take care if using SSR because is is only run once when server starts)
- what is inside function is run on each page load
